google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/GetAccountHierarchy.java

```diff
@@ -114,11 +122,14 @@ public static void main(String[] args) {
    * @param managerId the root customer ID from which to begin the search.
    * @param loginCustomerId the loginCustomerId used to create the GoogleAdsClient.
    */
-  private void runExample(GoogleAdsClient googleAdsClient, Long managerId, Long loginCustomerId) {
+  private void runExample(GoogleAdsClient googleAdsClient, Long managerId, long loginCustomerId)
```

Comment:
Why do you change the type to long, whereas the managerId variable is still the same?
Besides, could you check with Josh or Nick?
I remember vaguely that there is a reason why we declared this using Long, not long.

Reply:
Our approach has been:
* If it's potentially null, then use boxed `Long`
* Otherwise, use `long`